db/seeds.rb

```diff
@@ -470,6 +472,21 @@ def create_if_none(klass, count = nil)
 
 ##############################################################################
 
+num_page_redirects = 2 * SEEDS_MULTIPLIER
+
+seeder.create_if_none(PageRedirect, num_page_redirects) do
+  articles_for_old_paths = Article.where(published: true).order(Arel.sql("RANDOM()")).limit(num_page_redirects)
+  articles_for_new_paths = Article.where.not(id: articles_for_old_paths.map(&:id), published: false).order(Arel.sql("RANDOM()")).limit(num_page_redirects)
+
+  articles_for_old_paths.each_with_index do |old_article, i|
+    new_article = articles_for_new_paths[i]
+    PageRedirect.create!(old_path: old_article.path, new_path: new_article.path)
+  end
+end
+
+##############################################################################
+
+# rubocop:disable Rails/Output
```

Comment:
I'm having second thoughts about including this in the seed file. I could see it being confusing in development when a couple of random articles always redirect. Any thoughts?